src/components/app/details/triggerView/workflow.service.ts

```diff
@@ -57,6 +94,9 @@ export function processWorkflow(
     let ciPipelineToNodeWithDimension = (ciPipeline: CiPipeline) => ciPipelineToNode(ciPipeline, dimensions)
     const filteredCIPipelines =
         ciResponse?.ciPipelines?.filter((pipeline) => pipeline.active && !pipeline.deleted) ?? []
+    const configuredMaterialList = new Map<string, Set<string>>()
```

Comment:
move this to `handleSourceNotConfigured`

Reply:
done